tests/fixtures/MediaTest/create_check_file_changes.json

```diff
@@ -10,6 +10,8 @@
             "name": "file.png",
             "is_public": false,
             "owner_id": 2,
+            "preview_name": "file_preview.png",
```

Comment:
?

Reply:
This fixture remains from the previous implementation. I have deleted it now.